package.json

```diff
@@ -52,7 +52,8 @@
     "lodash": "^4.17.21",
     "luxon": "^3.2.1",
     "require-css": "^0.1.10",
-    "vue": "^3.4.15"
+    "vue": "^3.4.15",
+    "vue-router": "^4.3.0"
```

Comment:
I think is `vue-router` must be a peer depndancies. what do you think @blaisefonseca

Reply:
@sbilodeau  I don't think we even need it. @tahmeedshah  we should use tab and load the main vue file from the folder. there is no need for router here.

Reply:
Removed the router and added tabs to load preview pages for each section.